src/index.ts

```diff
@@ -1,44 +1,30 @@
 import { app, BrowserWindow, ipcMain, IpcMain } from 'electron';
-import storage from 'electron-json-storage';
 import { join } from 'path';
-import { of } from 'rxjs';
-import usbDetection from 'usb-detection';
-import registerManageDeviceSubscriptionHandler from './ipc/device-subscription';
 import registerShowOpenDialog from './ipc/show-open-dialog';
 import registerShowSaveDialog from './ipc/show-save-dialog';
 import registerFileSystemWriteFileHandler from './ipc/file-system-write-file';
-import registerGetBatteryInfoHandler from './ipc/get-battery-info';
-import registerGetPrinterInfoHandler from './ipc/get-printer-info';
-import registerPrintHandler from './ipc/print';
-import registerPrinterSubscription from './ipc/printer-subscription';
-import registerPrintToPDFHandler from './ipc/printToPDF';
 import registerQuitHandler from './ipc/quit';
 import registerSaveAsHandler from './ipc/saveAs';
 import registerLogHandler from './ipc/log';
 import registerCaptureScreenshot from './ipc/capture-screenshot';
 import parseOptions, { printHelp } from './utils/options';
-import autoconfigurePrint from './utils/printing/autoconfigurePrinter';
 import { getMainScreen } from './utils/screen';
-import { USBDetectionManager } from './utils/usb';
 import { HandlerContext } from './handlerContext';
 
 export type RegisterIpcHandler = (
   ipcMain: IpcMain,
-  { options, changedDevices, autoconfiguredPrinter }: HandlerContext,
+  { options }: HandlerContext,
 ) => (() => void) | void;
 
 // Allow use of `speechSynthesis` API.
 app.commandLine.appendSwitch('enable-speech-dispatcher');
 
-// Set storage location for user data.
-storage.setDataPath(join(app.getPath('userData'), 'data'));
-
```

Comment:
We don't need `storage` and haven't for a while, since all state is in the app backends now.